gnosis/safe-react
# Fetch safe apps list from the api

## What it solves

Closes #2282 

## How this PR fixes it

It fetches the Safe Apps list from the new API

## How to test it

- Open the Apps section
- Add a custom App
- Check if apps are retrieved
- If the endpoint fails, it should show the user their locally loaded apps
